### PR TITLE
Add userSlice & userInfo redux state

### PR DESCRIPTION
**Pull request by soyeongmin**

### CODE REVIEW

File: src/state/features/userSlice.ts [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+type userInfoType = {
+  name: string
+  email: string
+  img?: string
+  isLoggedIn: boolean
+}
+
+const initialState: userInfoType = {
+  name: '',
+  email: '',
```
Comment (jhw123) [inline src/state/features/userSlice.ts:12]:
```suggestion
  email: '',
  img: '',
```
make it consistent with logout

File: src/App.tsx [hunk 2]
```diff
@@ -1,63 +1,62 @@
...
 
   useEffect(() => {
-      if (!isLoggedIn) navigate("/login");
-      console.log(isLoggedIn)
-  }, [isLoggedIn])
-
-  function loginout(state: boolean){
-    setIsLoggedIn(state);
-  }
+    console.log(userInfo)
```
Comment (jhw123) [inline src/App.tsx:40]: Let's remove our test code in PR!

File: src/state/reducers/index.ts [hunk 3]
```diff
@@ -0,0 +1,6 @@
+import { combineReducers } from 'redux'
+import errorReducer from '../features/errorSlice'
+
+export const reducers = combineReducers({
```
Comment (jhw123) [inline src/state/reducers/index.ts:4]: is `reducers` ever used? If not, please remove it

**Review by jhw123 — CHANGES_REQUESTED** [review]

File: src/App.tsx (hunk 2 above)
Comment (soyeongmin) [reply src/App.tsx:40]: yep!

File: src/state/reducers/index.ts (hunk 3 above)
Comment (soyeongmin) [reply src/state/reducers/index.ts:4]: hmm I don't know. But We can use it by editing store.ts.

**Comment by soyeongmin** [comment]
Edit some codes (store, reducers, userSlice)
Delete console.log codes
Check whether the userInfo really change

**Review by jhw123 — APPROVED** [review]

**Pull request merged**